airbyte-integrations/bases/base-singer/base_singer/singer_helpers.py

```diff
@@ -151,9 +152,13 @@ def get_catalogs(logger, shell_command: str, sync_mode_overrides: Dict[str, Sync
             shell_command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True
         )
 
+        has_error = False
         for line in completed_process.stderr.splitlines():
             logger.log_by_prefix(line, "ERROR")
+            has_error = True
 
+        if has_error and not completed_process.stdout:
```

Comment:
Why was this piece added?

If you want to do this you'll need to also bump version on base-singer and update the FROM statement in the connector docker file

Reply:
it prevents from raising an uninformative exception if we have no output in stdout - I will update the base-singer

Reply:
Make sure to publish base-singer too